docs/modules/ROOT/pages/tmail-backend/jmap-extensions/folderFilteringAction.adoc

```diff
@@ -0,0 +1,236 @@
+= Folder filtering tasks
+:navtitle: folder filtering tasks
+
+This extension allows users to run their configured filtering rules (as defined by `Filter/set`) against existing messages in a specific mailbox.
+This is done by creating a `FolderFilteringAction` task.
+
```

Comment:
Please say that `com:linagora:params:jmap:filter` is mandatory to use FolderFilteringAction

Please include doc for `com:linagora:params:jmap:filter` versionning and explain that v2 exposes support for `FolderFilteringAction`

Why not put this in the https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/pages/tmail-backend/jmap-extensions/jmapFilters.adoc as this is technically the same JMAP capability ?